query.go

```diff
@@ -129,6 +135,14 @@ func DoneChan(doneChan chan<- struct{}) QueryOption {
 	}
 }
 
+// PersistToDisk allows the caller to tell that the filter should be kept
+// on disk once it's found.
+func PersistToDisk(persistToDisk bool) QueryOption {
```

Comment:
Why does this need a parameter? The default should be false, so users would only use `PersistToDisk()`.